app/api/v2/models/store_model.py

```diff
@@ -56,6 +56,19 @@ def update_product(self, product_id, product_name, category_id, stock_amount, pr
             return {"Message": "Product Updated successfully"}, 201
         except Exception as e:
             print(e)
+    
+    def update_stock_amount(self, product_name, stock_amount):
+        """Update Product."""
+        database = self.db
+        try:
+            curr = database.cursor()
+            query = "UPDATE products SET stock_amount=%s WHERE product_name=%s;"
+            curr.execute(query, (stock_amount,product_name))
+            database.commit()
+            curr.close()
+            return {"Message": "Product Updated successfully"}, 201
+        except Exception as e:
+            print(e)
```

Comment:
Exceptions should be returned rather than printed

Reply:
Okay thanks I will work on it